packages/flutter/lib/src/rendering/sliver_persistent_header.dart

```diff
@@ -365,20 +361,11 @@ abstract class RenderSliverScrollingPersistentHeader extends RenderSliverPersist
     return stretchOffset > 0 ? 0.0 : math.min(0.0, paintExtent - childExtent);
   }
 
-
   @override
   void performLayout() {
     final SliverConstraints constraints = this.constraints;
     final double maxExtent = this.maxExtent;
     layoutChild(constraints.scrollOffset, maxExtent);
```

Comment:
Optional: This could really just be simplified without the local variables:

```suggestion
    layoutChild(constraints.scrollOffset, maxExtent);
```

Reply:
What a nice change.